code/__DEFINES/misc.dm

```diff
@@ -255,10 +255,11 @@
                             0, 0, 1, 0,\
                             0, 0, 0, 1)
 
-#define MATRIX_GREYSCALE list(0.3, 0.3, 0.3, 0,\
-                              0.3, 0.3, 0.3, 0,\
-                              0.3, 0.3, 0.3, 0,\
-                              0,   0,   0,   1)
+#define MATRIX_GREYSCALE list(0.33, 0.33, 0.33, 0,\
```

Comment:
Ah, that checks out. I forgot that the alpha channel made the base color matrix 4x4, with the offset column making it a 5x5!